Test fail, exocore binary: PR #212 + #213 + #206
1. launch single validator localnet.
2. enable oracle feeding price.
3. mock deposit NST with EOA account.
4. check staker asset info
asset_infos:
```
- asset_id: 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee_0x65
  info:
    pending_undelegation_amount: "0"
    total_deposit_amount: "32000000000000000000"
    withdrawable_amount: "32000000000000000000"
- asset_id: 0x0000000000000000000000000000000000000000_0x0
  info:
    pending_undelegation_amount: "0"
    total_deposit_amount: "0"
    withdrawable_amount: "0"
```
5. then export, it shows below errors: 
```
panic: key of stakerInfo must be construct by 2 infos: assetID/stakerAddr

goroutine 130 [running]:
github.com/ExocoreNetwork/exocore/x/oracle/types.ParseNativeTokenStakerKey({0xc000086e1c?, 0xc002901b60?, 0xc002918880?})
        github.com/ExocoreNetwork/exocore/x/oracle/types/key_native_token.go:40 +0x71
github.com/ExocoreNetwork/exocore/x/oracle/keeper.Keeper.GetAllStakerInfosAssets({{0x441c3a0, 0xc00177b7b0}, {0x43d77f0, 0xc000821680}, {0x43d7890, 0xc000821760}, {{0x441c3a0, 0xc00177b7b0}, 0xc000523308, {0x43d77f0, ...}, ...}, ...}, ...)
        github.com/ExocoreNetwork/exocore/x/oracle/keeper/native_token.go:81 +0x1ce
github.com/ExocoreNetwork/exocore/x/oracle.ExportGenesis({{0x43ff0d0, 0x63b04c0}, {0x441cb80, 0xc0004e7800}, {{0x0, 0x0}, {0x0, 0x0}, 0x95, {0x0, ...}, ...}, ...}, ...)
        github.com/ExocoreNetwork/exocore/x/oracle/genesis.go:71 +0x5bb
github.com/ExocoreNetwork/exocore/x/oracle.AppModule.ExportGenesis({{{0x441c3a0, 0xc00177b7b0}}, {{0x441c3a0, 0xc00177b7b0}, {0x43d77f0, 0xc000821680}, {0x43d7890, 0xc000821760}, {{0x441c3a0, 0xc00177b7b0}, ...}, ...}, ...}, ...)
        github.com/ExocoreNetwork/exocore/x/oracle/module.go:147 +0x98
github.com/cosmos/cosmos-sdk/types/module.(*Manager).ExportGenesisForModules.func1({0x7f08f450f968, 0xc001153ea0}, 0xc00010e5a0)
        github.com/cosmos/cosmos-sdk@v0.47.8/types/module/module.go:402 +0xf1
created by github.com/cosmos/cosmos-sdk/types/module.(*Manager).ExportGenesisForModules in goroutine 1
        github.com/cosmos/cosmos-sdk@v0.47.8/types/module/module.go:400 +0x36e
```

--- a/x/oracle/keeper/native_token.go
+++ b/x/oracle/keeper/native_token.go
@@ -9,6 +9,7 @@ import (
 	utils "github.com/ExocoreNetwork/exocore/utils"
 	assetstypes "github.com/ExocoreNetwork/exocore/x/assets/types"
 	"github.com/ExocoreNetwork/exocore/x/oracle/types"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common/hexutil"
 )
@@ -63,8 +64,9 @@ func (k Keeper) GetStakerInfos(ctx sdk.Context, assetID string) (ret []*types.St
 // GetAllStakerInfosAssets returns all stakerInfos combined with assetIDs they belong to, used for genesisstate exporting
 func (k Keeper) GetAllStakerInfosAssets(ctx sdk.Context) (ret []types.StakerInfosAssets) {
 	store := ctx.KVStore(k.storeKey)
+	store = prefix.NewStore(store, types.NativeTokenStakerKeyPrefix(""))
 	// set assetID as "" to iterate all value with different assetIDs
-	iterator := sdk.KVStorePrefixIterator(store, types.NativeTokenStakerKeyPrefix(""))
+	iterator := sdk.KVStorePrefixIterator(store, []byte{})
 	defer iterator.Close()
 	ret = make([]types.StakerInfosAssets, 0)
 	l := 0

--- a/x/oracle/keeper/native_token.go
+++ b/x/oracle/keeper/native_token.go
@@ -21,8 +21,6 @@ import (
 
 const maxEffectiveBalance = 32
 
-var stakerList types.StakerList
-
 // SetStakerInfos set stakerInfos for the specific assetID
 func (k Keeper) SetStakerInfos(ctx sdk.Context, assetID string, stakerInfos []*types.StakerInfo) {
 	store := ctx.KVStore(k.storeKey)
@@ -94,8 +92,6 @@ func (k Keeper) SetStakerList(ctx sdk.Context, assetID string, sl *types.StakerL
 	store := ctx.KVStore(k.storeKey)
 	bz := k.cdc.MustMarshal(sl)
 	store.Set(types.NativeTokenStakerListKey(assetID), bz)
-	// set cache used by updateNSTByBalanceChange
-	stakerList = *sl
 }
 
 // GetStakerList return stakerList for native-restaking asset of assetID
@@ -174,6 +170,7 @@ func (k Keeper) UpdateNSTValidatorListForStaker(ctx sdk.Context, assetID, staker
 
 	keyStakerList := types.NativeTokenStakerListKey(assetID)
 	valueStakerList := store.Get(keyStakerList)
+	var stakerList types.StakerList
 	stakerList.StakerAddrs = make([]string, 0, 1)
 	if valueStakerList != nil {
 		k.cdc.MustUnmarshal(valueStakerList, &stakerList)
@@ -231,7 +228,7 @@ func (k Keeper) UpdateNSTByBalanceChange(ctx sdk.Context, assetID string, rawDat
 	if len(rawData) < 32 {
 		return errors.New("length of indicate maps for stakers shoule be exactly 32 bytes")
 	}
-	sl := k.getStakerList(ctx, assetID)
+	sl := k.GetStakerList(ctx, assetID)
 	if len(sl.StakerAddrs) == 0 {
 		return errors.New("staker list is empty")
 	}
@@ -287,14 +284,6 @@ func (k Keeper) UpdateNSTByBalanceChange(ctx sdk.Context, assetID string, rawDat
 		store.Set(key, bz)
 	}
 	return nil
-}
-
-// getStakerList returns all Stakers for native-restaking of assetID, this is used for cache
-func (k Keeper) getStakerList(ctx sdk.Context, assetID string) types.StakerList {
-	if len(stakerList.StakerAddrs) == 0 {
-		stakerList = k.GetStakerList(ctx, assetID)
-	}
-	return stakerList
 }
 
 // parseBalanceChange parses rawData to details of amount change for all stakers relative to native restaking

